redash/utils/__init__.py

```diff
@@ -86,8 +90,8 @@ def process_default(self, o):
         return False, None  # default processing
 
 
-class JSONEncoder(JSONEncoderMixin, json.JSONEncoder):
-    """Adapter for `json.dumps`."""
+class JSONEncoder(JSONEncoderMixin, simplejson.JSONEncoder):
+    """Adapter for `simplejson.dumps`."""
```

Comment:
I think, if one of JSONEncoder classes is removed, JSONEncoderMixin can also be removed and code from `JSONEncoderMixin.process_default` can be moved to `JSONEncoder.default`

Reply:
Good idea, I didn't know the history of the class. Will change..